includes/wf_crm_webform_ajax.inc

```diff
@@ -180,7 +184,7 @@ class wf_crm_webform_ajax extends wf_crm_webform_base {
       return TRUE;
     };
 
-    if (!empty($webform->getElementDecoded($fid)['#private'])) {
+    if (!empty($webform->getElement($fid)['#private'])) {
```

Comment:
This is possibly a false positive because default fields are never present. Review.